GSA/made-in-america
# [MIA-159]: includes feature to download csvs

closes https://cm-jira.usa.gov/browse/MIA-159

## Description

- Sets a file as Blob  and assigns it as ObjectURL to our download button
- Includes Cypress test includes checking on the download element to confirm the file is actually downloaded

## How to test

- Verify cypress test is passing in pipeline
- Navigate to the [waiver page](https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/preview/gsa/made-in-america/feature/csv-download/waivers/)
- Click "Download .CSV File" button
- Confirm file downloaded is named `waivers.csv`
- Open `waivers.csv` file on your local machine
- Confirm that cells are filled with our [data](https://github.com/GSA/made-in-america-data/blob/main/waivers.csv)

To prepare for code review, please follow the following checks.
